src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+const Category = () => (
+  <div>
+    <h1>Categories</h1>
```

Comment:
- [ ] [OPTIONAL] As written in the project requirements, your categories page should display only a button with text "Check status". It's currently empty

![image](https://user-images.githubusercontent.com/52934128/184236215-26f3d808-aecf-4c8d-8edb-eeb84f2d4b39.png)